django/db/backends/postgresql/base.py

```diff
@@ -94,7 +100,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         "PositiveBigIntegerField": "bigint",
         "PositiveIntegerField": "integer",
         "PositiveSmallIntegerField": "smallint",
-        "SlugField": "varchar(%(max_length)s)",
+        "SlugField": _get_varchar_column,
```

Comment:
As far as I'm aware, slug should be short and descriptive. I don't think that providing an option for unlimited slug is a step forward.

Reply:
anecdotally I've seen quite a few crazy slugs (but that's probably the exception rather than the norm), but I think the same argument as above applies here.